dll/win32/shell32/stubs.cpp

```diff
@@ -1028,24 +1028,11 @@ FirstUserLogon(LPWSTR lpUnknown1, LPWSTR lpUnknown2)
  */
 EXTERN_C HRESULT
 WINAPI
-SHSetFolderPathA(int csidl,
-                 HANDLE hToken,
-                 DWORD dwFlags,
-                 LPCSTR pszPath)
-{
-    FIXME("SHSetFolderPathA() stub\n");
-    return E_FAIL;
-}
-
-/*
- * Unimplemented
- */
-EXTERN_C HRESULT
-WINAPI
-SHSetFolderPathW(int csidl,
-                 HANDLE hToken,
-                 DWORD dwFlags,
-                 LPCWSTR pszPath)
+SHSetFolderPathW(
+    _In_ INT csidl,
+    _In_ HANDLE hToken,
+    _In_ DWORD dwFlags,
+    _In_z_ LPCWSTR pszPath)
```

Comment:
Aren't LP(C)(W)STR typedefs automatically annotated to be null-terminated? (so that the `_z` thing in the annotation is unnecessary?)

Reply:
Fixed.